crates/nomination/src/ext.rs

```diff
@@ -39,8 +39,16 @@ pub(crate) mod vault_registry {
     pub use frame_support::dispatch::{DispatchError, DispatchResult};
     pub use vault_registry::{DefaultVault, VaultStatus};
 
-    pub fn get_backing_collateral<T: crate::Config>(vault_id: &T::AccountId) -> Result<Collateral<T>, DispatchError> {
-        <vault_registry::Pallet<T>>::get_backing_collateral(vault_id)
+    // excluding nomination
+    pub fn get_vault_collateral<T: crate::Config>(vault_id: &T::AccountId) -> Result<Collateral<T>, DispatchError> {
```

Comment:
```suggestion
    pub fn get_vault_only_collateral<T: crate::Config>(vault_id: &T::AccountId) -> Result<Collateral<T>, DispatchError> {
```
Maybe more explicit that way?